### PR TITLE
Feature/11 filter by number range

### PR DESCRIPTION
**Pull request by jimibilibob**

### CODE REVIEW

File: Pokedex_ios/View/RangeSlider/RangeSlider.swift [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+    let trackLayer = RangeSliderTrackLayer() // = CALayer() defined in RangeSliderTrackLayer.swift
+    let lowerThumbLayer = RangeSliderThumbLayer() // CALayer()
+    let upperThumbLayer = RangeSliderThumbLayer() // CALayer()
+    let lowerThumbLabel = UILabel()
+    let upperThumbLabel = UILabel()
+    var previousLocation = CGPoint()
+
+    var trackTintColor = UIColor(white: 0.9, alpha: 1.0)
+    var trackHighlightTintColor = UIColor(named: "apply-button")
```
Comment (jmacboy) [inline Pokedex_ios/View/RangeSlider/RangeSlider.swift]: This color already has a constant value for the apply-button string, please reuse that

File: Pokedex_ios/View/RangeSlider/RangeSliderThumbLayer.swift [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            ctx.fillPath()
+
+            // Outline
+            ctx.setStrokeColor(shadowColor!.cgColor)
+            ctx.setLineWidth(3) // Thumb stroke width
+            ctx.addPath(thumbPath.cgPath)
+            ctx.strokePath()
+
+            // Add a shadow when highlighted
+            /*if highlighted {
```
Comment (jmacboy) [inline Pokedex_ios/View/RangeSlider/RangeSliderThumbLayer.swift]: Don’t add commented code if it won’t be needed on the project

**Review by jmacboy — CHANGES_REQUESTED** [review]

File: Pokedex_ios/View/RangeSlider/RangeSlider.swift (hunk 1 above)
Comment (jimibilibob) [reply Pokedex_ios/View/RangeSlider/RangeSlider.swift]: Now using the constant value.

File: Pokedex_ios/View/RangeSlider/RangeSliderThumbLayer.swift (hunk 2 above)
Comment (jimibilibob) [reply Pokedex_ios/View/RangeSlider/RangeSliderThumbLayer.swift]: Removed unnecessary code

**Review by jmacboy — APPROVED** [review]

**Pull request merged**